### DISCUSSION TITLE
adding `clear()` method to `useListData`?

### DISCUSSION THREAD

**Discussion by stefanprobst** [adobe/react-spectrum#5979] [Ideas]
hi, i am currently using RAC's `<Form>` component, together with `useListData` as a stand-in for a `useFieldArray` hook.

this is working great, the only thing i have found missing is an easy way to clear/reset `useListData`.

would you accept a pr which adds that?

```ts
clear() {
  dispatch(state => {
    return {
      ...state,
      items: [],
      selectedKeys: new Set()
    }
  })
}
```

**Comment by CurryYangxx** [comment]
+1
